web/src/javascript/leaderboardApp.js

```diff
@@ -0,0 +1,16 @@
+import {fetchEntries} from "./leaderboardEntry";
+
+function displayEntries(entries) {
+  const leaderboard = document.getElementById("leaderboard-body");
+  for(const entry of entries) {
+    const row = leaderboard.insertRow(0);
+    row.insertCell(0).innerHTML = entry.team_id;
+    row.insertCell(1).innerHTML = entry.name;
```

Comment:
I know that we said that we won't focus on security, but could we encode the HTTP entities here? :) I *really* don't want to see a team called `<img src='/' onerror="alert('trololo');">` during the competition. :P